Source/WebCore/html/HTMLFormControlElement.cpp

```diff
@@ -395,14 +395,19 @@ void HTMLFormControlElement::setPopoverTargetAction(const AtomString& value)
 }
 
 // https://html.spec.whatwg.org/#popover-target-attribute-activation-behavior
-void HTMLFormControlElement::handlePopoverTargetAction() const
+void HTMLFormControlElement::handlePopoverTargetAction(const EventTarget* eventTarget) const
 {
     RefPtr target = popoverTargetElement();
     if (!target)
         return;
 
     ASSERT(target->popoverData());
 
+    if (RefPtr eventTargetElement = dynamicDowncast<Node>(eventTarget)) {
+        if (target->containsIncludingShadowDOM(eventTargetElement.get()) && target.get()->isDescendantOrShadowDescendantOf(this))
```

Comment:
```suggestion
        if (target->containsIncludingShadowDOM(eventTargetElement) && target->isDescendantOrShadowDescendantOf(this))
```

You shouldn't need to use `.get()` here.

Reply:
It seems I do need eventTargetElement.get() (see below), but I don't need ~~target~~popover.get() so I'll remove that.

```
/Source/WebCore/html/HTMLFormControlElement.cpp:407:49: error: no viable conversion from
      'RefPtr<const Node>' (aka 'WTF::RefPtr<const WebCore::Node>') to 'const Node *'
  407 |         if (popover->containsIncludingShadowDOM(eventTargetNode) && popover->isDescendantOrShadowDescendantOf(this))
      |                                                 ^~~~~~~~~~~~~~~
...
/Users/luke/Development/BrowserEngines/WebKit/Source/WebCore/dom/Node.h:444:63: note: passing argument to parameter here
  444 |     WEBCORE_EXPORT bool containsIncludingShadowDOM(const Node*) const;
  ```